Packs/SaviyntEIC/Integrations/SaviyntEICEventCollector/SaviyntEICEventCollector.yml

```diff
@@ -0,0 +1,98 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: SaviyntEICEventCollector
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+  section: Connect
+- display: Analytics Name
+  name: analytics_name
+  required: true
+  type: 16
+  defaultvalue: SIEMAuditLogs
+  options:
+  - SIEMAuditLogs
+  section: Collect
+- defaultvalue: '50000'
+  section: Collect
+  display: Maximum number of events per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+- name: eventFetchInterval
+  section: Collect
+  display: Events Fetch Interval
+  advanced: true
+  defaultvalue: "1"
+  type: 19
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+description: Collector for Saviynt Enterprise Identity Cloud (EIC) audit logs using Analytics Runtime Control V2.
+display: Saviynt EIC Event Collector
+name: SaviyntEICEventCollector
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: 'false'
+      description: If true, the command will create events, otherwise it will only display them.
+      isArray: false
+      name: should_push_events
+      predefined:
+      - 'true'
+      - 'false'
+      required: true
+    - default: false
+      description: Maximum number of results to return (max 10000).
+      isArray: false
+      name: limit
+      required: true
+      secret: false
+    - default: false
+      description: Time frame in minutes back from now to query.
+      isArray: false
+      name: time_frame
+      required: false
+      secret: false
+    - default: false
+      description: Offset for paging.
+      isArray: false
+      name: offset
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets events from Saviynt EIC.
```

Comment:
```suggestion
    description: Gets events from Saviynt EIC. This command is supported in Cortex XSIAM only and is intended to be used for debugging purposes as it may result in duplicate events.
```

Reply:
Done.